pypeit/spectrographs/keck_deimos.py

```diff
@@ -225,28 +227,43 @@ def get_detector_par(self, det, hdu=None):
             # get measurements
             tab_measure = Table.read(measure_files[close_idx], format='ascii')
             measured_det = tab_measure['col3']
+            measured_amptype = tab_measure['col4']
             measured_gain = tab_measure['col5']  # [e-/DN]
             measured_ronoise = tab_measure['col7']   # [e-]
-            msgs.info(f"We are using DEIMOS gain/RN values based on WMKO estimates on {measure_dates[close_idx]}.")
+            msgs.info(f"We are using DEIMOS gain/RN values for AMPMODE = {amp} "
+                      f"based on WMKO estimates on {measure_dates[close_idx]}.")
+            # find values for this amp and each detector
+            this_amp = measured_amptype == 'A' if amp == 'SINGLE:A' else measured_amptype == 'B'
```

Comment:
Same here. An exception for `'DUAL:A+B'`.

Reply:
Since I added an error message above, here is not necessary.